Source/interfac.cpp

```diff
@@ -188,6 +189,21 @@ void DrawCutscene()
 
 } // namespace
 
+Rectangle UIRectangle;
+const Rectangle &GetUIRectangle()
+{
+	return UIRectangle;
+}
+void CalculateUIRectangle()
+{
+	constexpr int UIWidth = 640;
+	constexpr int UIHeight = 480;
+	UIRectangle = {
+		{ (gnScreenWidth - UIWidth) / 2, (gnScreenHeight - UIHeight) / 2 },
+		{ UIWidth, UIHeight }
+	};
+}
+

```

Comment:
Despite the friendly name of `interface.cpp` this file only deals with the load screens. This should probably live in `control.cpp` instead.

Reply:
Initially I did it here, cause `UI_OFFSET_Y` was also definied in `interfac.h` 😉 
Currently `GetUIRectangle` is used for menu UI and ingame UI.
That's why I wouldn't move it to `control.cpp` or `diablui.cpp`.

We could
- Move it to `ui_fwd.h`/`display.cpp` like `GetScreenWidth`
- Separate menu UI and ingame UI size (`GetIngameUIRectangle` and `GetMenuUIRectangle` ?)
- Leave it for a later refactoring
- Something else? 😛

Reply:
Looks like I made the mistake back in https://github.com/diasurgical/devilutionX/commit/f450d6a125966ef12ecc0aa8bcb23627768a74ba

I guess move it to ui_fwd.h for now

Reply:
Done. 🙂